Merge Home and Profile pages into develop

highwir3 opened the pull request.

highwir3 committed:
Initial commit

highwir3 committed:
Merge branch 'develop' into home

highwir3 committed:
Added popular courses

seccion de cursos populares y refactorizacion de algunas cosas

highwir3 committed:
Added video courses and latest blog sliders to home, template my training slider in profile

highwir3 committed:
added yoga categories placeholder, fixed home header

highwir3 committed:
Added Transloco to home and profile pages, fixed some formatting

highwir3 committed:
added circular bar component and implemented it in home

highwir3 committed:
Merge branch 'develop' into home

highwir3 committed:
Revert "Merge branch 'develop' into home"

This reverts commit a6bd59b97232d04441b9fea1b1f76a44cddf839d, reversing
changes made to 6d0dd30c3aa9db6e893bf3464dc451852d6a51ba.

highwir3 committed:
added chart in profile and transloco, fixed dark mode circular bar

install ngx charts

highwir3 committed:
added progress bar in profile, fixed dark mode chart

highwir3 committed:
Merge branch 'develop' into home

highwir3 committed:
Revert "Merge branch 'develop' into home"

This reverts commit 7d2d836758557df4e5119a445ab256ccb7ac49e8, reversing
changes made to 21a19a7edbf0952dc15747e1a997d5d7d667d1dc.

highwir3 committed:
Reapply "Merge branch 'develop' into home"

This reverts commit a0c38b4dc8c193eab582bd71f7b4674ebf0612ca.

highwir3 committed:
Reapply "Merge branch 'develop' into home"

This reverts commit eb1219c08f34b19c25483ab79da09cbe5e42c9b9.

highwir3 committed:
Update auth.service.ts

highwir3 committed:
Re-installed ngx charts

RumbosE reviewed (dismissed):
Tu rama no esta actualizada Israel. la cosa es que se acaba de hacer en merge a develop. Usa las interfaces y adaptadores que estan en sus respectivas carpetas y  si nececitas nuevas interfaces para el profile agregalas en la carpeta interfaces vamos a limpiar un poquito mas lo modulos

highwir3 committed:
Merge branch 'develop' into home

paulgamboag commented:
Eduardo realizó cambios en algunos de los componentes del Home. Las interfaces son diferentes, por lo que debes actualizar tu rama, y ajustar los datos para poder usar el componente compartido. Por ejemplo: <app-carrusel-bg-img [courses]="popularCourses"/>. Sería simplemente realizar ese ajuste (que los datos de popularCourses sigan la interfaz), del resto. Todo bien.

Eduardo también creó una carpeta con algunos adaptadores, puede ser que te sirvan, caso contrario, simplemente ajusta los datos a la interfaz.

highwir3 committed:
fixed adapters for home and profile

highwir3 committed:
Added My Training service in profile

RumbosE reviewed (commented):
Me parece que quedaron bien las interfaces.
Ciertas cositas y detalles de css que siento que podrian mejorar, pero por ahora siento que pasa la prueba.

Sin embarjo creo que deberias contemplar que los datos que obtienes del usuario que muestras en la pagina home y de profile vienen de un servicio que deberias hacer referencia en tu component.ts y no pintarlo nada mas en el html.

Las inteface que creaste en home-page.component. siento que deberian estar en la carpeta de Interfaces de home para que se puedan modificar o usar en toda la aplicacion de ser necesario.

E importante que los id acepten string ya que el backend esta haciendo uso de los UUID

highwir3 committed:
Added interfaces to home folder, added user info service

Agregue las interfaces que tenia en home a la carpeta de inferfaces, y cree un servicio getUserInfo para la informacion del usuario dentro de home y profile

paulgamboag reviewed (commented):
Tomado en cuenta el modo oscuro y la traducción. Muy bien.

En las categorías de yoga de la página de Home, pienso que se pudo colocar íconos y textos en vez de una imagen en gris.

Próximamente se hará la integración con el backend, y habrá que refactorizar partes de la aplicación. En general, bien hecho. Bastante parecido al diseño original

paulgamboag committed:
Merge branch 'develop' into home

RumbosE reviewed (approved):
fino 🍕

paulgamboag reviewed (approved):
Bien hecho
